modules/nf-core/rtgtools/rocplot/tests/main.nf.test

```diff
@@ -0,0 +1,69 @@
+nextflow_process {
+
+    name "Test Process RTGTOOLS_ROCPLOT"
+    script "../main.nf"
+    process "RTGTOOLS_ROCPLOT"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "rtgtools"
+    tag "rtgtools/rocplot"
+
+    test("homo_sapiens - tsv") {
+
+        setup {
+            run("UNTAR") {
+                script "../../../untar/main.nf"
```

Comment:
```suggestion
                script "modules/nf-core/untar/main.nf"
```
Probably more robust and a bit clearer with an absolute path since these two modules are unrelated.

Reply:
I followed the steps detailed here: https://hackmd.io/@nf-core/ryWsFLwyT I'm not sure what's the best option, pinging @sateeshperi for assistance :grin:

Reply:
For reference, in this module an absolute path was used: https://github.com/nf-core/modules/blob/master/modules/nf-core/sratools/fasterqdump/tests/main.nf.test

Reply:
we are going to stick with absolute paths for now to keep it consistent.....I will update the fasterqdump module

Reply:
So absolute path for dependency modules and relative path for the module that actually needs testing?